README.md

```diff
@@ -1,26 +1,34 @@
-OpenAPI 3.x or Swagger 2.0? YAML or JSON? URL, path, string or object? Who cares! It just works.
+<p align="center">
+  <a href="https://npm.im/oas-normalize">
+    <img src="https://user-images.githubusercontent.com/33762/200143162-237bb893-3cda-40d6-a1f6-edd5b8f978db.png" alt="oas-normalize" />
```

Comment:
idk about this right now because I just could not find a transparent Swagger logo that wasn't 300x300 so it's a bit... chunky around the edges.

![Screen Shot 2022-11-05 at 3 36 08 PM](https://user-images.githubusercontent.com/33762/200144072-e6d2c341-8988-46b2-91b7-ef5d2b1784ac.png)

Reply:
I feel like we should have some sort of ReadMe iconography in this — I see a lot of OAS libraries in the wild that use generic logos like this.